code/__DEFINES/xeno.dm

```diff
@@ -72,6 +72,12 @@ GLOBAL_LIST_INIT(xeno_ai_spawnable, list(
 	/mob/living/carbon/xenomorph/scorpion/ai,
 ))
 
+///Checks if xeno has damage
+#define XENO_IS_DAMAGED(xeno) (xeno.health < xeno.maxHealth)
+
+///Checks if xeno has damage or missing overheal
+#define XENO_IS_DAMAGED_FULL(xeno) (XENO_IS_DAMAGED(xeno) || xeno.overheal < xeno.xeno_caste.overheal_max)
```

Comment:
these defines are not needed, they just make the code cryptic

Reply:
removed